stable/sysdig/templates/clusterrolebinding.yaml

```diff
@@ -15,4 +15,5 @@ subjects:
 roleRef:
   kind: ClusterRole
   name: {{ template "sysdig.fullname" .}}
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Good catch!

If we are going to add the apiGroup change to the ClusterRoleBinding, please could you add the apiGroup key also to subject?

You can read more about it here: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding

Reply:
@nestorsalceda - it looks like the ServiceAccount doesn't require apiGroup - and in fact when I tried to add it I ended up with and error : 

```
Error: release sysdig-agent failed: ClusterRoleBinding.rbac.authorization.k8s.io "sysdig-agent" is invalid: subjects[0].apiGroup: Unsupported value: "rbac.authorization.k8s.io": supported values: ""
```